apps/studio/components/to-be-cleaned/Dropdown/OrganizationDropdown.tsx

```diff
@@ -21,36 +22,38 @@ const OrganizationDropdown = ({
   const router = useRouter()
 
   const organizationCreationEnabled = useIsFeatureEnabled('organizations:create')
+  const { isSuccess: orgsLoaded } = useOrganizationsQuery()
 
   return (
-    <DropdownMenu>
-      <DropdownMenuTrigger>
-        <Button asChild>
-          <span>New project</span>
+    <div className="flex gap-4">
+      <DropdownMenu>
+        <DropdownMenuTrigger>
+          <Button asChild>
+            <span>New project</span>
+          </Button>
+        </DropdownMenuTrigger>
+        <DropdownMenuContent side="bottom" align="center">
+          <>
+            <DropdownMenuLabel>Choose organization</DropdownMenuLabel>
+            {organizations
+              .sort((a, b) => a.name.localeCompare(b.name))
+              .map((x) => (
+                <DropdownMenuItem key={x.slug} onClick={() => router.push(`/new/${x.slug}`)}>
+                  {x.name}
+                </DropdownMenuItem>
+              ))}
+          </>
+        </DropdownMenuContent>
+      </DropdownMenu>
+
+      {IS_PLATFORM && organizationCreationEnabled && orgsLoaded && organizations.length !== 0 && (
```

Comment:
Should this be:
```suggestion
      {IS_PLATFORM && organizationCreationEnabled && orgsLoaded && organizations.length === 0 && (
```
?

Reply:
Is this button even needed then?